File: Logica C/log4.c
#include <stdio.h>

int main(void) {
  int N, i, cont = 0;

  printf("Informe 15 valores inteiros: ");

  for (i = 0; i < 15; i++) {
    scanf("%d", &N); //guarda o valor digitado pelo usuário após o início da estrutura de repetição
    if (N >= 10 && N <= 20) {
      cont++;
    }
  }

  printf("Foram informados %d valores no intervalo de 10 a 20 especificado.\n", cont);

  return 0;
}

File: Logica JS/log4.js
let n;
let i;
let cont = 0;
let outputElement = document.getElementById('output');

outputElement.innerHTML += "Informe 15 valores inteiros: <br>";

for (i=0; i< 15; i++){
    n = prompt("Informe os valores: ");

    if (n >= 10 && n <= 20){
        cont++;
    }
}

outputElement.innerHTML += "Foram informados "+cont+" valores no intervalo de 10 a 20.";